json_asset.py

```diff
@@ -1,29 +1,24 @@
 import requests
 from datetime import datetime, timezone, timedelta
+import bisect
 
 def current_time():
     JST = timezone(timedelta(hours=+9), 'JST')
 
-    frequency = [3, 8, 13, 18, 23, 28, 33, 38, 43, 48, 53, 58]
-    now_minute = datetime.now(JST).minute
-
-    for i, minute in enumerate(frequency):
-        if now_minute in [0, 1, 2, 3]:
-            preformatted = datetime.now(JST) + timedelta(minutes=-5)
-            current_time = str(preformatted.strftime('%Y%m%d%H')) + str(frequency[-1]-1).zfill(2)
-            break
-        elif now_minute in [58, 59]:
-            current_time = str(datetime.now(JST).strftime('%Y%m%d%H')) + str(frequency[-1]-1).zfill(2)
-            break
-        elif now_minute > minute: #3:12
-            continue
+    def helper(now_minute=datetime.now(JST).minute):
```

Comment:
Do we need a nested function? Or we could just write it as a part of the 
```python
def current_time()
```

Reply:
True, I didn't realise I could have just assigned the value to some property i.e. fetched_time 😂